src/overlay/Overlay.js

```diff
@@ -83,6 +83,7 @@ Overlay.defaultProps = {
   overlayBackgroundColor: 'white',
   width: windowWidth - 80,
   height: windowHeight - 180,
+  onBackdropPress: () => {},
```

Comment:
This will fail eslint since it's an empty function body. That's why I suggested you use `() => null`. See https://github.com/react-native-training/react-native-elements/blob/master/src/searchbar/SearchBar-ios.js#L172 for example

Reply:
makes sense